releases/8.0.0.md

```diff
@@ -57,6 +58,8 @@ JavaScript:
 
 - `ensure_userprofile_exists()` was removed from the `Tile` model.
 
+- The `arches` object is no longer available in frontend components that exist outside of defined Arches patterns. If your project contains Vue components that are mounted on templates that inherit directly from `base-root.htm`, they will need to be updated to use `generateArchesURL` and the settings API instead.
```

Comment:
Suggest something a little more specific than "defined patterns", which makes sense to people who read on about `base-root.htm` and know what that is, but others might need help understanding if this applies to them.